tests/unittests/test_builtin_handlers.py

```diff
@@ -473,4 +474,28 @@ def test_render_jinja_payload_replaces_missing_variables_and_warns(self):
         self.assertIn(expected_log, self.logs.getvalue())
 
 
+class TestShellScriptByFrequencyHandlers(CiTestCase):
```

Comment:
The `CiTestCase` base class isn't needed here. You can just remove it along with the `with_logs = True` on the following line.

Reply:
Done